src/config.rs

```diff
@@ -304,195 +304,203 @@ impl Config {
         )
-        .arg(Arg::with_name("max-object-size")
+        .arg(Arg::new("max-object-size")
             .long("max-object-size")
             .value_name("BYTES")
+            .action(ArgAction::Set)
             .help("Maximum size of downloaded objects (0 for no limit)")
-            .takes_value(true)
         )
-        .arg(Arg::with_name("enable-bgpsec")
+        .arg(Arg::new("max-ca-depth")
+            .long("max-ca-depth")
+            .value_name("COUNT")
+            .action(ArgAction::Set)
+            .help("Maximum distance of a CA from a trust anchor")
+        )
+        .arg(Arg::new("enable-bgpsec")
             .long("enable-bgpsec")
+            .action(ArgAction::SetTrue)
             .help("Include BGPsec router keys in the data set")
         )
-        .arg(Arg::with_name("dirty-repository")
+        .arg(Arg::new("dirty-repository")
             .long("dirty")
+            .action(ArgAction::SetTrue)
             .help("Do not clean up repository directory after validation")
         )
-        .arg(Arg::with_name("validation-threads")
-             .long("validation-threads")
-             .value_name("COUNT")
-             .help("Number of threads for validation")
-             .takes_value(true)
+        .arg(Arg::new("validation-threads")
+            .long("validation-threads")
+            .value_name("COUNT")
+            .action(ArgAction::Set)
+            .help("Number of threads for validation")
         )
-        .arg(Arg::with_name("verbose")
-             .short("v")
-             .long("verbose")
-             .multiple(true)
-             .help("Log more information, twice for even more")
+        .arg(Arg::new("verbose")
+            .short('v')
+            .long("verbose")
+            .action(ArgAction::Count)
+            .help("Log more information, twice for even more")
         )
-        .arg(Arg::with_name("quiet")
-             .short("q")
-             .long("quiet")
-             .multiple(true)
-             .conflicts_with("verbose")
-             .help("Log less information, twice for no information")
+        .arg(Arg::new("quiet")
+            .short('q')
+            .long("quiet")
+            .action(ArgAction::Count)
+            .conflicts_with("verbose")
+            .help("Log less information, twice for no information")
         )
-        .arg(Arg::with_name("syslog")
+        .arg(Arg::new("syslog")
              .long("syslog")
+            .action(ArgAction::SetTrue)
              .help("Log to syslog")
         )
-        .arg(Arg::with_name("syslog-facility")
-             .long("syslog-facility")
-             .takes_value(true)
-             .default_value("daemon")
-             .help("Facility to use for syslog logging")
+        .arg(Arg::new("syslog-facility")
+            .long("syslog-facility")
+            .default_value("daemon")
+            .action(ArgAction::Set)
+            .help("Facility to use for syslog logging")
```

Comment:
The default value is not mentioned in this help message, is that on purpose?

Reply:
It is actually added by the help generator thingamajig:
```
      --syslog-facility <FACILITY>
          Facility to use for syslog logging [default: daemon]
```